schemas/test/en/test_list_collector_list_summary.json

```diff
@@ -111,6 +121,202 @@
                     "id": "group",
                     "title": "Questions",
                     "blocks": [
+                        {
+                            "id": "introduction",
+                            "type": "Introduction",
```

Comment:
If we are adding one, it should be very basic, this has so much content that isn't needed.

Reply:
It is very basic now, literally one sentence/link on the page!